inductiva/fluids/scenarios/wind_tunnel/post_processing.py

```diff
@@ -24,96 +22,128 @@
         """Get aerodynamics data over an object inside the WindTunnel.
 
         Current Support - OpenFOAM
         """
 
-        reading_file = os.path.join(self.sim_output_path, "foam.foam")
+        # The OpenFOAM data reader from PyVista requires that a file named
+        # "foam.foam" exists in the simulation output directory.
+        # Create this file if it does not exist.
+        foam_file_path = os.path.join(self.sim_output_path, "foam.foam")
+        pathlib.Path(foam_file_path).touch(exist_ok=True)
 
-        # Create reading file
-        with open(reading_file, "w", encoding="utf-8") as file:
-            file.close()
+        reader = pv.OpenFOAMReader(foam_file_path)
+        reading_file = os.path.join(self.sim_output_path, "foam.foam")
 
         # Initialize reader and define reading time-step
         reader = pv.POpenFOAMReader(reading_file)
-        reader.set_active_time_value(self.time_step)
+        reader.set_active_time_value(time_step)
 
-        mesh = reader.read()
-        object_data = mesh["boundary"]["object"]
+        full_mesh = reader.read()
+        domain_mesh = full_mesh["internalMesh"]
+        object_mesh = full_mesh["boundary"]["object"]
 
-        return object_data
+        return domain_mesh, object_mesh
 
-    def get_physical_field(self, physical_property: str = "pressure"):
+    def get_physical_field(self,
+                           physical_property: str = "pressure",
+                           time_step: float = 50):
         """Get a physical scalar field over mesh points for a certain time_step.
 
         Returns:
             A MeshData object that allow to manipulate the data over a mesh
             and to render it.
         """
+
+        _, object_mesh = self.get_mesh_data(time_step)
+
         property_notation = OpenFOAMPhysicalProperty[
             physical_property.upper()].value
-        physical_field = MeshData(self.object_data, property_notation)
+        physical_field = MeshData(object_mesh, property_notation)
 
         return physical_field
 
     def get_streamlines(self,
-                        physical_property: Literal["pressure",
-                                                   "velocity"] = "pressure"):
-        """Get streamlines over the object in the WindTunnel."""
+                        time_step: float = 50,
+                        max_time: float = 100,
+                        n_points: int = 100,
+                        initial_step_length: float = 1,
+                        source_radius: float = 0.7,
+                        save_path: Path = None):
+        """Get streamlines over the object in the WindTunnel.
+        
+        The streamlines are obtained by seeding a set of points
+        at the inlet of the WindTunnel.
+
+        Args:
+            time_step: Simulation time step to obtain the streamlines.
+            max_time: Maximum time for the streamlines.
```

Comment:
Maximum time of the simulation? What does this mean?

Reply:
I will change it.